Compile bug: can't build with MSVC 17.13.0

### Git commit

e598697d63d062b4af939f1b0383d6adc6292d1a

### Operating systems

Windows

### GGML backends

CUDA

### Problem description & steps to reproduce

Hi, I updated MSVC to 17.13.0 and Windows SDK to 26100.3037, CUDA Toolkit version is 12.8.0. I have found that the #include <chrono> header is missing in 3 files: log.cpp, perplexity.cpp, imatrix.cpp
So I just added it to all 3 files and everything compiled successfully

### First Bad Commit

_No response_

### Compile command

```shell
cmake -B build -DGGML_CUDA=ON
cmake --build build --config Release -j 6
```

### Relevant log output

```shell
F:\LLM\llama.cpp-master\common\log.cpp(18,79): error C2039: 'system_clock': is not a member of 'std::chr
ono' [F:\LLM\llama.cpp-master\build\common\common.vcxproj]
      C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.43.34808\include\__msvc_c
  hrono.hpp(286,11):
      see declaration of 'std::chrono'

F:\LLM\llama.cpp-master\common\log.cpp(18,79): error C3083: 'system_clock': the symbol to the left of a
'::' must be a type [F:\LLM\llama.cpp-master\build\common\common.vcxproj]
F:\LLM\llama.cpp-master\common\log.cpp(18,93): error C2039: 'now': is not a member of 'std::chrono' [F:\
LLM\llama.cpp-master\build\common\common.vcxproj]
      C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.43.34808\include\__msvc_c
  hrono.hpp(286,11):
      see declaration of 'std::chrono'

F:\LLM\llama.cpp-master\common\log.cpp(18,93): error C3861: 'now': identifier not found [F:\LLM\llama.cp
p-master\build\common\common.vcxproj]
F:\LLM\llama.cpp-master\common\log.cpp(18,25): error C2672: 'std::chrono::duration_cast': no matching ov
erloaded function found [F:\LLM\llama.cpp-master\build\common\common.vcxproj]
      C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.43.34808\include\__msvc_c
  hrono.hpp(87,19):
      could be '_To std::chrono::duration_cast(const std::chrono::duration<_Rep,_Period> &) noexcept(<ex
  pr>)'
```

--- a/common/log.cpp
+++ b/common/log.cpp
@@ -1,5 +1,6 @@
 #include "log.h"
 
+#include <chrono>
 #include <condition_variable>
 #include <cstdarg>
 #include <cstdio>

--- a/examples/imatrix/imatrix.cpp
+++ b/examples/imatrix/imatrix.cpp
@@ -3,6 +3,7 @@
 #include "log.h"
 #include "llama.h"
 
+#include <chrono>
 #include <cmath>
 #include <cstdio>
 #include <cstring>

--- a/examples/perplexity/perplexity.cpp
+++ b/examples/perplexity/perplexity.cpp
@@ -3,6 +3,7 @@
 #include "log.h"
 #include "llama.h"
 
+#include <chrono>
 #include <algorithm>
 #include <array>
 #include <atomic>

